helm_deploy/templates/_helpers.tpl

```diff
@@ -50,6 +50,18 @@ app.kubernetes.io/name: {{ include "laa-submit-crime-forms.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
 {{- end }}
 
+{{/*
+Default Security Context
+*/}}
+{{- define "helm_deploy.defaultSecurityContext" -}}
```

Comment:
for consistency it should be prefixed with the what other "methods" are prefixed with, namely 

```
{{- define "laa-submit-crime-forms.defaultSecurityContext" -}}
```

Reply:
Although we are already using a combination, as at line 81 and 89.

Reply:
yeah I've updated it throughout now so it's consistent and all are using laa-submit-crime-forms (I think that's clearer because helm_deploy gives off the impression we are using in built helm stuff and not custom stuff)